install/gcp/deployment_manager/istio-cluster.yaml

```diff
@@ -9,12 +9,15 @@ resources:
     gkeClusterName: istio-cluster
     zone: us-central1-a
     initialNodeCount: 4
-    instanceType: n1-standard-1
+    instanceType: n1-standard-2
     enableAutomaticSidecarInjection: true
     enableMutualTLS: true
     enablePrometheus: true
     enableGrafana: true
     enableTracing: true
-    enableServiceGraph: true
+    enableServiceGraph: false
+    enableKiali: true
+    kialiUsername: admin
+    kialiPassword: mysecret
```

Comment:
Everywhere else, we call the kiali credentials a "passphrase" (due to terminology that the Istio team wants everyone to use). I suggest, to avoid confusion and maintain consistency, to rename this "kialiPassphrase".